dev/devicelab/lib/framework/cocoon.dart

```diff
@@ -66,7 +66,7 @@ class Cocoon {
 
   /// Parse the local repo for the current running branch.
   String _readCommitBranch() {
-    final ProcessResult result = processRunSync('git', <String>['rev-parse', '--abbrev-ref', 'HEAD']);
+    final ProcessResult result = processRunSync('git', <String>['branch', '--show-current']);
```

Comment:
The devicelab tests aren't as robust, and we inject a static response on the test cases. Filed https://github.com/flutter/flutter/issues/71855 to track separately.

This is to unblock the DeviceLab to LUCI migration that the infra team is working on this week.